README.md

```diff
@@ -81,17 +81,19 @@ Note that the icons in this set are tied to be used in [Calypso](https://github.
 
 This icon set uses a few automation scripts to ease the generation of new icons in a reliable way. In short, we require `node` and `grunt`. For detailed instructions check [the installation page](https://github.com/Automattic/gridicons/wiki/Installation).
 
+Once you checkout the repo run `npm install` in the `gridicons` folder. 
+To generate all the fonts, svgs and so on you run `npm run build` 
```

Comment:
These details should already be covered in the other lists. What is clarified by adding these lines?

Reply:
There was no where on the page that tells the user to run `npm install` after checkout the repo. Maybe there needs to be a new sections here explains how to get started adding a new icon.  This part was supposed to help the user get started.

Reply:
It's in the link right above the two lines you added: https://github.com/Automattic/gridicons/wiki/Installation

But: message received. Now I understand what it went wrong. :D